lib/cancel/CanceledError.js

```diff
@@ -8,10 +8,12 @@ var utils = require('../utils');
  *
  * @class
  * @param {string=} message The message.
+ * @param {Object=} config The config.
+ * @param {Object=} request The request.
  */
-function CanceledError(message) {
-  // eslint-disable-next-line no-eq-null,eqeqeq
-  AxiosError.call(this, message == null ? 'canceled' : message, AxiosError.ERR_CANCELED);
+function CanceledError(message, config, request) {
+  // eslint-disable-next-line no-eq-null
+  AxiosError.call(this, message == null ? 'canceled' : message, AxiosError.ERR_CANCELED, config, request);
```

Comment:
Please use strict equal

Reply:
This was in this way before I changed it. I assume that is to check against `undefined` and `null`, but not empty strings, `0` or other "falsy" values.

Reply:
My mistake, I see,  please just add back this in line 15:

```
// eslint-disable-next-line no-eq-null,eqeqeq
````

Reply:
That should resolve the build issue

Reply:
Done!